setup.py

```diff
@@ -16,31 +17,29 @@
 
 if sys.version_info[0] < 3:
     import ConfigParser as configparser
+
     open_kwargs = {}
 else:
     import configparser
-    open_kwargs = {'encoding':'utf-8'}
+
+    open_kwargs = {'encoding': 'utf-8'}
+
 
 def check_hdf5version(hdf5_includedir):
     try:
-        f = open(os.path.join(hdf5_includedir,'H5pubconf-64.h'),**open_kwargs)
+        f = open(os.path.join(hdf5_includedir, 'H5public.h'), **open_kwargs)
     except IOError:
-        try:
-            f = open(os.path.join(hdf5_includedir,'H5pubconf-32.h'),**open_kwargs)
-        except IOError:
-            try:
-                f = open(os.path.join(hdf5_includedir,'H5pubconf.h'),**open_kwargs)
-            except IOError:
-                return None
+        return None
     hdf5_version = None
     for line in f:
-        if line.startswith('#define H5_VERSION'):
-            hdf5_version = line.split()[2]
+        if line.startswith('#define H5_VERS_INFO'):
+            hdf5_version = line.split('"')[1]
```

Comment:
looks like another weirdness injected for pep'ifying :(  I'm now worried about this tool